### ISSUE TITLE
Unexpected label ***

### ISSUE THREAD

**Issue by jadeqian** [cdmh/deeplab-public#3]
I followed the instruction at https://bitbucket.org/deeplab/deeplab-public/ to train Pascal VOC2012 dataset and got error.

F1007 16:26:39.428997 17015 softmax_loss_layer.cpp:85] Unexpected label 220
*** Check failure stack trace: ***
    @     0x7f85a4788daa  (unknown)
    @     0x7f85a4788ce4  (unknown)
    @     0x7f85a47886e6  (unknown)
    @     0x7f85a478b687  (unknown)
    @           0x50d643  caffe::SoftmaxWithLossLayer<>::Forward_cpu()
    @           0x473066  caffe::Net<>::ForwardFromTo()
    @           0x4734a7  caffe::Net<>::ForwardPrefilled()
    @           0x4a6d6f  caffe::Solver<>::Solve()
    @           0x418ff2  train()
    @           0x412021  main
    @     0x7f85a1472ec5  (unknown)
    @           0x417a27  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)

I followed FAQ at http://ccvl.stat.ucla.edu/deeplab_faq/ and added “ignore_label: 255″ to both softmax_loss_layer and seg_accuracy_layer" in train.prototxt . Still cannot solve the problem. Anybody can help me? thanks in advance.

**Comment by agatabcool** [comment]
Did you ever figure this error out? I added ignore_label: 220 and it just pops up new unfound labels?

**Comment by cdmh** [comment]
This repo is a clone of the original authors. Have you tried asking at https://bitbucket.org/deeplab/deeplab-public/ ?

**Comment by agatabcool** [comment]
Yes, I read that on your other questions- my question was directed towards jadeqian. 

Just for the sake of having the answer on the internet - this code uses bilinear interpolation to downsize images which causes these unexpected labels. The solution is to change pixels with labels 255 to either 0, 22, or the label of the surrounding objects.
